### PR TITLE
doc/rados: edit operations/add-or-rm-mons (2 of x)

### PR DESCRIPTION
**Pull request by zdover23**
Edit the second part of doc/rados/operations/add-or-rm-mons.rst.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/rados/operations/add-or-rm-mons.rst [hunk 1]
```diff
@@ -217,43 +217,44 @@ quorum.
...
 
       monmaptool /tmp/monmap --rm b
       monmaptool /tmp/monmap --rm c
-	
-#. Inject the surviving map with the removed monitors into the
-   surviving monitor(s).  For example, to inject a map into monitor
-   ``mon.a``, follow the example below:
+
+#. Inject the surviving map that includes the removed monitors into the
+   surviving monitor(s):
```
Comment (anthonyeleven) [inline doc/rados/operations/add-or-rm-mons.rst:235]: surviving monitor(s) monmap:

File: doc/rados/operations/add-or-rm-mons.rst [hunk 2]
```diff
@@ -217,43 +217,44 @@ quorum.
...
+
 #. Start only the surviving monitors.
 
-#. Verify the monitors form a quorum (``ceph -s``).
+#. Verify that the monitors form a quorum by running the command ``ceph -s``.
+
+#. The data directory of the removed monitors is in ``/var/lib/ceph/mon``:
+   either archive this data directory in a safe location or delete this data
+   directory.  However, do not delete it unless you are confident that the
+   remaining monitors are healthy and sufficiently redundant.
```
Comment (anthonyeleven) [inline doc/rados/operations/add-or-rm-mons.rst:256]: Maybe add a note about ensuring that there is sufficient room for expansion and compaction of the live DB in addition to an archived copy, and possibly compressing the latter.

File: doc/rados/operations/add-or-rm-mons.rst [hunk 3]
```diff
@@ -262,183 +263,190 @@ Changing a Monitor's IP Address
...
 .. important:: Existing monitors are not supposed to change their IP addresses.
 
-Monitors are critical components of a Ceph cluster, and they need to maintain a
-quorum for the whole system to work properly. To establish a quorum, the
-monitors need to discover each other. Ceph has strict requirements for
-discovering monitors.
+Monitors are critical components of a Ceph cluster. The entire system can work
+properly only if the monitors maintain quorum, and quorum can be established
+only if the monitors have discovered each other by means of their IP addresses.
+Ceph has strict requirements on the discovery of monitors.
```
Comment (anthonyeleven) [inline doc/rados/operations/add-or-rm-mons.rst:269]: Tempted to replace this entire section with the word *DON'T*.

**Review by anthonyeleven — APPROVED** [review]
auggestions

File: doc/rados/operations/add-or-rm-mons.rst (hunk 1 above)
Comment (zdover23) [reply doc/rados/operations/add-or-rm-mons.rst:235]: Added.

File: doc/rados/operations/add-or-rm-mons.rst (hunk 2 above)
Comment (zdover23) [reply doc/rados/operations/add-or-rm-mons.rst:256]: Added.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/52825 - Reef backport 
https://github.com/ceph/ceph/pull/52826 - Quincy backport

**Pull request merged**